nnvm/python/nnvm/frontend/darknet.py

```diff
@@ -850,6 +870,31 @@ def _handle_darknet_rnn_layers(self, layer_num, sym):
 
         return processed, sym
 
+    def _make_outlist(self, sym, op_name, layer, layer_num):
+        if layer.type == LAYERTYPE.REGION:
+            l_param = np.array([layer.n, layer.out_c, layer.out_h, layer.out_w,
+                                layer.classes, layer.coords, layer.background],
+                               dtype=np.int32)
+            self._outs.append(_sym.Variable(name='region'+str(layer_num)+'_attribute',
+                                            init=l_param))
+            k = self._get_tvm_params_name(op_name, 'bias')
+            self._outs.append(_sym.Variable(name=k, init=self._tvmparams[k].asnumpy()))
```

Comment:
Why always the region layer outputs are appended to the out symbol? There can be some cases where region can be an intermediate layer.. according to this logic, it will b always part of out symbol. 
Same issue for yolo layer also.

Reply:
Darknet yolo models takes all the yolo layer and region layer outputs for postprocessing. Darknet haven't provided any way to select outputs. So as far i thought of appending all yolo and region output.